src/ansys/fluent/core/solver/flobject.py

```diff
@@ -153,10 +153,12 @@ def get_attr(self, attr, attr_type_or_types=None) -> Any:
             val = attrs[attr]
 
         if attr_type_or_types:
-            if not type(attr_type_or_types) == tuple:
+            if not isinstance(attr_type_or_types, tuple):
```

Comment:
I feel we should avoid passing tuple here. A given attribute of a given type should be of definite datatype. This will also make type casting easier if we decide to extend it for other datatypes.